flocker/node/agents/ebs.py

```diff
@@ -289,16 +334,12 @@ def __init__(self, blockdevice_id, operation,
 
 
 class EliotLogHandler(logging.Handler):
-    # Whitelist ``"msg": "Params:`` field for logging.
-    _to_log = {"Params"}
-
     def emit(self, record):
-        fields = vars(record)
         # Only log certain things.  The log is massively too verbose
         # otherwise.
-        if fields.get("msg", ":").split(":")[0] in self._to_log:
+        if record.levelname == 'INFO':
```

Comment:
This means you won't log things with e.g. ERROR level. Since you already limit to INFO or higher below you can just omit this if and log everything.